src/ccl/charts/charts-container.component.ts

```diff
@@ -22,7 +23,7 @@ import { Chart } from './chart.component';
 export class Charts extends OnInit {
 
     private chartList: Array<String>;
-    private chartData: Object;
+    private chartData: Object<ChartData>;
```

Comment:
Can this just be `ChartData` instead of `Object<ChartData>` in all these places? I'm not sure using the object generic gains anything as every class inherits from object anyways.